artifact.json

```diff
@@ -0,0 +1,11 @@
+{
+  "projectName": "targeting:braze-components",
+  "vcsURL": "https://github.com/guardian/braze-components",
+  "actions": [
+    {
+      "action": "storybook-static",
```

Comment:
Don't forget to change this if we change the name in the RiffRaff.yaml file as well.